modules/multi-architecture-creating-arm64-bootimage.adoc

```diff
@@ -5,9 +5,9 @@
 :_mod-docs-content-type: PROCEDURE
 [id="multi-architecture-creating-arm64-bootimage_{context}"]
 
-= Creating an ARM64 boot image using the Azure image gallery
+= Creating a 64-bit ARM or 64-bit AMD boot image using the Azure image gallery
```

Comment:
```suggestion
= Creating a 64-bit ARM or 64-bit x86 boot image using the Azure image gallery
```

I see the other references for amd64 are 64-bit x86.

Reply:
Yes! Updated it to x86